src/components/BoardWrapper.jsx

```diff
@@ -37,12 +37,27 @@ export const BoardWrapper = (props) => {
     return <div>Generating Board...</div>;
   }
 
+  // For puzzle mode, is first covered by a black cover that we must click to reveal it
+  const hideCover = () => {
+    document.getElementById("board-cover").classList.add("hidden");
```

Comment:
generally considered bad practice to use document.getelementbyid in react, should use refs instead. you should be able to setup a ref similar to how timerRef is setup, then for board-container div, set ref={your_ref}. then in your function you should be able to do something like your_ref.current.classList.add('hidden')

Reply:
you may also want to, instead of adding hidden classname, have some state that causes a conditional render of either the cover or the board. because right now, i can easily just go into devtools and remove the cover and see the puzzle without starting the timer.